test/popd.js

```diff
@@ -116,11 +116,30 @@ test('Test that rootDir is not stored', t => {
   t.truthy(shell.error());
 });
 
-test('quiet mode', t => {
+test('quiet mode off', t => {
   try {
     shell.config.silent = false;
+    shell.pushd('test/resources/pushd');
```

Comment:
I think this should be moved up one line, otherwise it will print during the test (which we try to avoid). Same thing for the `quite mode on` test.